docs/cloud-runner/06-development.md

```diff
@@ -0,0 +1,96 @@
+| stream editor changes             | todo   | Game CI will stream any local changes to a build machine.           |
+| send uncommitted git changes      | todo   | Run automated jobs without having to commit, discover errors early. |
+| warm starts and push hooks        | todo   | Do not have to wait for cloud runner to spin up a new machine.      |
+| track target git reference by sha | todo   | Target very specific points in git history.                         |
+```
+
+## Backlog
+
+```md
+| What                                  | Status | Why                                                       |
+| ------------------------------------- | ------ | --------------------------------------------------------- |
+| resilience hooks                      | todo   | Retry, startup and cleanup hooks.                         |
+| custom logging                        | todo   | Support extensible logging to your own loggin service.    |
+| custom storage engine                 | todo   | Share Unity import cache between local and runners.       |
+| better artifact handling              | todo   | Better extensibility and support for artifact handling.   |
+| network game features                 | todo   | Quick deploy simulation server, relay or headless client. |
+| interactive cloud runner              | todo   |                                                           |
+| scriptable custom provider            | todo   |                                                           |
+| cleaned up debug/prod                 | todo   |                                                           |
+| custom cloud runner docker image      | todo   |                                                           |
+| infographics for cloud runner docs    | todo   |                                                           |
+| detect incoming binary conflicts      | todo   |                                                           |
+| test coverage for hooks and steps     | todo   |                                                           |
+| multiplayer test sample               | todo   |                                                           |
+| code coverage report / project report | todo   |                                                           |
+| kuberenetes Rook support              | todo   |                                                           |
+| custom provider sample                | todo   |                                                           |
+| easily find all error lines in a log  | todo   |                                                           |
+```
+
+Minor changes, suggestions and bugs are tracked as GitHub issues:
+[Game CI Issues - GitHub](https://github.com/game-ci/unity-builder/labels/cloud-runner)
+
+
+# Contributions
+
+Source contributors are welcome via PR. You can always discuss a suggestion on discord.
+
+Donations welcome.
```

Comment:
I would argue that this is not a message for individual features. Recommend removing entirely and optionally adding a section to the homepage if you like.

The readmes already contain this information and the contribution guides have been set up for all repos.